infra/vercel/main.tf

```diff
@@ -0,0 +1,27 @@
+import {
+  to = vercel_project.monoweb
+  id = "prj_ojfJBhSht8MDWDP1gxeTRAa6vV75"
+}
+
+resource "vercel_project" "monoweb" {
+  name      = "monoweb"
+  framework = "nextjs"
+
+  git_repository = {
+    production_branch = "main"
+    type              = "github"
+    repo              = "dotkom/monoweb"
+  }
+
+  build_command  = "cd ../.. &&  pnpm build:web"
+  root_directory = "apps/web"
+
+  lifecycle {
+    prevent_destroy = true
```

Comment:
This way we don't accidentally delete the vercel project when taking down staging or something weird